vcd_cli/profile.py

```diff
@@ -26,3 +29,31 @@ def profile(ctx):
     """
     profiles = Profiles.load()
     click.echo(yaml.dump(profiles.data, default_flow_style=False))
+
+
+@vcd.command(short_help='current resources in use')
+@click.pass_context
+def pwd(ctx):
```

Comment:
To me, profile sounds better than pwd.

Reply:
we already have `profile` to show current content of the profile file. `profile` will be used in the future as a group command and will have `list`, `info`, `use`, `create`, etc subcommands. But I agree in that we can try to find a better alternative.